lib/feedsme/index.js

```diff
@@ -54,153 +64,223 @@ function pkgName(pkg) {
  * @param {Fucntion} done Completion callback.
  * @api private
  */
-Feedsme.prototype.change = function change(env, data, done) {
-  var pkg = this.extractLatest(data);
-  var app = this.app;
-  //
-  // TODO: Figure out if we can get previous here or if we need to do a GET on
-  // artifactory which would kind of suck
-  //
-
-  var meta = {
-    name: pkg.name,
-    version: pkg.version,
-    env: env
-  };
+Feedsme.prototype.change = async function change(env, data) {
+  const { Dependent, DependentOf } = this.models;
+  const pkg = this.extractLatest(data);
+  const publish = env === 'dev';
+
+  const { name, version } = pkg;
+
+  const meta = { name, version, env };
+
+  this.log.info(`Processing change for ${ pkgName(pkg) } in ${ env }`, meta);
+  // Update dependent and dependent_of models for reference
+  await this.resolve(env, pkg);
+
+  // Fetch things we need to figure out what exactly to do
+  const [releaseLine, dep, depOf] = await Promise.all([
+    publish ? this.release.head({ pkg: name }) : this.release.get({ pkg: name, version }),
+    Dependent.get(pkg.name),
+    DependentOf.get(pkg.name)
```

Comment:
can that return multiple things (e.g. package A is dependent of both package B and C)?

Reply:
@msluther the model only allows one value. We are basically only supporting the single parent package case

Reply:
I think then we definitely need to get to a manual parenting configuration soon then and not relying on package.json dependencies, as more things are built by wrhs the likelihood that there are accidental parentings by leaving something in dependencies rather than devDependencies increases.